run_tests.py

```diff
@@ -1,7 +1,9 @@
 import os
+import sys
 
 import pytest
 
 if __name__ == "__main__":
     os.environ['APP_SETTINGS'] = 'TestingConfig'
-    pytest.main(['--cov-report', 'term-missing', '--cov', 'ras_backstage'])
+    exitcode = pytest.main(['--cov-report', 'term-missing', '--cov', 'ras_backstage'])
+    sys.exit(exitcode)
```

Comment:
:+1:

Reply:
is there any value in running the tests via a Python module, rather than using the recommended test runner that comes with PyTest?

Reply:
I think this way was introduced in secure messaging because the tests ran both unit tests and the behave tests. Then it creeped it's way into other repos as we didn't use make files before, could just run it with the pytest command. I've just fixed the script to fail when tests fail here as it didn't do so previously. To remove the script is probably out of this tickets scope as there are at least three other repos like this.